### PR TITLE
Add time since NOD to intake stats page

### PR DESCRIPTION
**Pull request by leikkisa**
connects #4587 

### Description
Add time since NOD for Ramp Elections to intake stats page

### CODE REVIEW

**Comment by leikkisa** [comment]
Also want to check if this will break the stats page if we don't have the data backfilled first.

File: app/models/intake_stats.rb [hunk 1]
```diff
@@ -119,6 +119,13 @@ def intake_series_status(intakes)
       average(response_times)
     end,
 
+    average_election_nod_to_established_time: lambda do |range|
+      closed_appeals = RampClosedAppeal.joins(:ramp_election)
+        .where(ramp_elections: { receipt_date: offset_range(range) })
+      nod_times = closed_appeals.map { |c| c.established_at.beginning_of_day.to_f - c.nod_date.in_time_zone.to_f }
```
Comment (sansb) [inline app/models/intake_stats.rb]: This will have that "n+1" property I was talking about. Check out this tutorial- http://tomdallimore.com/blog/includes-vs-joins-in-rails-when-and-where/

tl;dr is you should probably replace `joins` with `includes`
